<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>231</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Encrypt</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>240</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Decrypt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>-10</y>
      <width>221</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Plain Text Encryption</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Text Input</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Encrypted Output</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>30</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter Key</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>130</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear Key</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>742</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen_Plain_Text"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_Encrypted_Text"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionEncrypt_Text"/>
    <addaction name="actionDecrypt_Text"/>
    <addaction name="separator"/>
    <addaction name="actionClear_Text"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_Encryption_Program"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="separator"/>
   <addaction name="actionOpen_Plain_Text"/>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="separator"/>
   <addaction name="actionSave_Encrypted_Text"/>
   <addaction name="separator"/>
   <addaction name="actionSave_as"/>
   <addaction name="separator"/>
   <addaction name="actionEncrypt_Text"/>
   <addaction name="separator"/>
   <addaction name="actionDecrypt_Text"/>
   <addaction name="separator"/>
   <addaction name="actionClear_Text"/>
  </widget>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="iconresource.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open Encrypted Text</string>
   </property>
  </action>
  <action name="actionSave_Encrypted_Text">
   <property name="icon">
    <iconset resource="iconresource.qrc">
     <normaloff>:/icons/saveas.png</normaloff>:/icons/saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save Encrypted Text</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="iconresource.qrc">
     <normaloff>:/icons/Save.png</normaloff>:/icons/Save.png</iconset>
   </property>
   <property name="text">
    <string>Save  as ...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="iconresource.qrc">
     <normaloff>:/icons/exit.png</normaloff>:/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_Encryption_Program">
   <property name="icon">
    <iconset resource="iconresource.qrc">
     <normaloff>:/icons/Very-Basic-About-icon.png</normaloff>:/icons/Very-Basic-About-icon.png</iconset>
   </property>
   <property name="text">
    <string>About Encryption Program</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="iconresource.qrc">
     <normaloff>:/icons/aboutqt.png</normaloff>:/icons/aboutqt.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionEncrypt_Text">
   <property name="icon">
    <iconset resource="iconresource.qrc">
     <normaloff>:/icons/Lock-Lock-icon.png</normaloff>:/icons/Lock-Lock-icon.png</iconset>
   </property>
   <property name="text">
    <string>Encrypt Text</string>
   </property>
  </action>
  <action name="actionDecrypt_Text">
   <property name="icon">
    <iconset resource="iconresource.qrc">
     <normaloff>:/icons/Lock-Unlock-icon.png</normaloff>:/icons/Lock-Unlock-icon.png</iconset>
   </property>
   <property name="text">
    <string>Decrypt Text</string>
   </property>
  </action>
  <action name="actionClear_Text">
   <property name="icon">
    <iconset resource="iconresource.qrc">
     <normaloff>:/icons/Clear-icon.png</normaloff>:/icons/Clear-icon.png</iconset>
   </property>
   <property name="text">
    <string>Clear Text</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="iconresource.qrc">
     <normaloff>:/icons/new.png</normaloff>:/icons/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen_Plain_Text">
   <property name="icon">
    <iconset resource="iconresource.qrc">
     <normaloff>:/icons/open-file-icon.png</normaloff>:/icons/open-file-icon.png</iconset>
   </property>
   <property name="text">
    <string>Open Plain Text</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="iconresource.qrc"/>
 </resources>
 <connections/>
</ui>
